packages/loader/container-loader/src/collabWindowTracker.ts

```diff
@@ -63,7 +63,7 @@ export class CollabWindowTracker {
         // We don't acknowledge no-ops to avoid acknowledgement cycles (i.e. ack the MSN
         // update, which updates the MSN, then ack the update, etc...). Also, don't
         // count system messages in ops count.
-        if (isSystemMessage(message)) {
+        if (!isClientMessage(message)) {
```

Comment:
I'd say similar to above, I'd use isRuntimeMessage here. Mostly because intent here is for runtime (and DDSs) not to keep too much tracking state / memory due to runtime ops from other clients. Summary op (from that angle) should be treated same as service ops - they do not impact client (or rather DDSs) directly.
Same - it's change in behavior, but I think it's good change.

Reply:
One clarification - I'd love to exclude summary op as well from this calculus, but it seems orthogonal / in the bucket "maybe we will do it later".
We have to use runtime check here (not client check) for different, more important reason - to avoid positive feedback loops with Noops, which are included into isClientMessage.

I'd hope some test fails with 0x0ce assert below with this change as is. If not, we should add test case here.